### ISSUE TITLE
Is it cause our app to be banned?

### ISSUE THREAD

**Issue by dawud-tan** [davethomas11/AndroidHotSwapCodeExample#1]
[an app may not download executable code (e.g. dex, JAR, .so files) from a source other than Google Play.](https://play.google.com/about/privacy-security-deception/malicious-behavior/#/about/privacy-security-deception/malicious-behavior/)

**Comment by Mohilvadia** [comment]
i don't think so as when you upload your app to google play store your app is checked thoroughly what does is contain and how it is working. By launching new app doesn't make your app malicious and code in that app will work accordingly. so don't worry
